app/styles/ember-frost-bunsen/frost-date-picker.scss

```diff
@@ -0,0 +1,8 @@
+.frost-bunsen-has-value {
```

Comment:
I don't know if this styling should go in `ember-frost-date-picker` or not. The default text it uses looks like a placeholder (to me at least!). We've been using this in quote.